### DISCUSSION TITLE
Magezon Page Builder

### DISCUSSION THREAD

**Discussion by eneskirca** [breezefront/community#102] [3. Integrations]
I am writing to inquire about the compatibility of your Magezon Page Builder extension with Breeze Theme.

As an e-commerce business owner, we have been using your Magezon Page Builder extension to easily create drag and drop landing pages on our website. We have been impressed with its features and functionality and have found it to be a valuable tool in our business.

However, we recently came across the Breeze Theme and are considering switching to this theme for our website. Before doing so, we wanted to confirm if your Magezon Page Builder extension is compatible with the Breeze Theme.

We would greatly appreciate any information or recommendations you can provide regarding this matter. We believe that the combination of your extension with the Breeze Theme could greatly benefit our business and improve our overall website performance.

**Comment by dimdimich** [comment]
Thank you for your interest. We already checked this extension and it has a lot of javascript code that make it's integration problematic. I would suggest to try contact Magezone and ask them to consider adding Breeze integration .

**Comment by vovayatsyuk** [comment]
Just to clarify: Why do you keep saying "**your** Magezon Page Builder extension"? It's not our extension. You are writing to Breeze Theme discussions.

p.s. We have an example of Pagebuilder module integration: https://github.com/breezefront/module-breeze-magezon-pagebuilder. You are welcome to work on other components.

**Reply by eneskirca** [reply]
it's my fault I know magezon p.b is not your extension, you say its integration is problematic but sharing magezon breeze version 
https://github.com/breezefront/module-breeze-magezon-pagebuilder 
Could u explain what you mean by it?

**Reply by vovayatsyuk** [reply]
This integration has one working component only: a collapsible widget. But the whole pagebuilder has hundreds of components.

**Comment by dimdimich** [comment]
We did some simple integration for several features of this module. Unfortunatly integrating entire module requires deep understanding of Magezone extension. So we would suggest to contact them.
